pkg/boot/grub/config_test.go

```diff
@@ -159,11 +139,37 @@ func FuzzParseGrubConfig(f *testing.F) {
 
 	f.Add([]byte("multiBoot 0\nmodule --nounzip"))
 	f.Fuzz(func(t *testing.T, data []byte) {
-		if len(data) > 1000000 {
+		if len(data) > 4096 {
+			return
+		}
+
+		// do not allow arbitrary files reads
+		if bytes.Contains(data, []byte("include")) {
 			return
 		}
 
-		err := os.WriteFile(path, data, 0o777)
+		baseDir := t.TempDir()
+
+		dirPath := baseDir + "/EFI/uefi"
```

Comment:
might as well fix this now: filepath.Join, not +
should we ever wish to run on windows, it ought to be, possibly:
filepath.Join(baseDir, "EFI", "uefi")

Reply:
fixed with ee962ad